Update instructions on hatch/VScode integration (#525)
scverse/cookiecutter-scverse: {{cookiecutter.project_name}}/docs/contributing.md

hatch run docs:build  # defined in the table [tool.hatch.envs.docs]
```

> ### VScode

Reviewer: ```suggestion
### VS Code
```

I think this is the official spelling

> If you are using VScode, install the [hatch-code][] extension.

Author: ```suggestion
If you are using VS code, install the [hatch-code][] extension.
```

Additionally, make sure that the `vscode-python-environments` extension is installed

> and `"python.useEnvironmentsExtension": true` is activated in your `settings.json` (both should be the default).

Reviewer: `"python.useEnvironmentsExtension": true` is not yet the default AFAIK

```suggestion
Additionally, make sure that the `vscode-python-environments` extension is installed (should be by default)
and `"python.useEnvironmentsExtension": true` is activated in your `settings.json`.
```
